docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -0,0 +1,70 @@
+# Archiveren (voor dummies)
+
+## Gerelateerde attributen
+
+* `Zaak.Archiefnominatie` Aanduiding of het zaakdossier blijvend bewaard of na een bepaalde termijn vernietigd moet worden:
+
+    * `null`
+    * `Blijvend bewaren`
+    * `Vernietigen`
+    
+* `Zaak.Archiefstatus` De fase waarin het zaakdossier zich qua archivering bevindt:
+
+    * `nog te archiveren` (standaardwaarde)
+    * `gearchiveerd`
+    * `gearchiveerd (procestermijn onbekend)`
+    * `vernietigd`
+    * `overgedragen`
+    
+  Als de `Archiefstatus` `gearchiveerd` of `gearchiveerd (procestermijn onbekend)` is, dan is het zaakdossier niet-wijzigbaar.
+
+* `Zaak.Archiefactiedatum` De datum waarop het gearchiveerde zaakdossier vernietigd moet worden dan wel overgebracht moet worden naar een archiefbewaarplaats. 
+
+* `ResultaatType.Archiefnominatie` Aanduiding die aangeeft of `Zaak`en met een resultaat van dit `ResultaatType` blijvend moeten worden bewaard of (op termijn) moeten worden vernietigd . 
+
+* `ResultaatType.Procestermijn` De periode dat het zaakdossier na afronding van de zaak actief gebruikt en/of geraadpleegd wordt ter ondersteuning van de taakuitoefening van de organisatie.
+
+* `ResultaatType.Archiefactietermijn` De termijn, na het vervallen van het bedrijfsvoeringsbelang, waarna het zaakdossier (de `Zaak` met alle bijbehorende `Informatieobject`en) van een `Zaak` met een resultaat van dit `ResultaatType` vernietigd of overgebracht (naar een archiefbewaarplaats) moet worden. 
+
+
+TODO:
+
```

Comment:
Brondatum opnemen

Reply:
`Brondatum` bestaat niet volgens mij, we hebben wel `ResultaatType.BrondatumArchiefprocedure`. Het doel van dit documentje is juist heel concreet en precies benoemen over welke attributen we het hebben.